pkg/crc/preflight/preflight_linux.go

```diff
@@ -166,22 +166,18 @@ func getPreflightChecksForDistro(distro *linux.OsRelease, networkMode network.Mo
 	checks := commonChecks()
 
 	if networkMode == network.VSockMode {
-		checks = append(checks, vsockPreflightChecks)
+		return append(checks, vsockPreflightChecks)
 	}
 
 	switch distroID(distro) {
-	case linux.Ubuntu:
-	case linux.RHEL, linux.CentOS, linux.Fedora:
-		if networkMode == network.DefaultMode {
-			checks = append(checks, nmPreflightChecks[:]...)
-			checks = append(checks, dnsmasqPreflightChecks[:]...)
-		}
 	default:
-		logging.Warnf("distribution-specific preflight checks are not implemented for %s", distroID(distro))
-		if networkMode == network.DefaultMode {
-			checks = append(checks, nmPreflightChecks[:]...)
-			checks = append(checks, dnsmasqPreflightChecks[:]...)
-		}
+		logging.Warnf("distribution-specific preflight checks are not implemented for '%s'", distroID(distro))
+		fallthrough
```

Comment:
I don't get this one, why we want to go to other part of switch if we don't support or implemented the distribution specific checks?

Reply:
This keeps the pre-existing behaviour, which is how crc has more or less always behaved, we try to configure NM+dnsmasq regardless of the distro. Once we are a little bit smarter about distribution detection (use `ID_LIKE` rather than `ID`), we can indeed justu ignore it.

Reply:
I tried to improve the commit log.